starter/script.js

```diff
@@ -1 +1,69 @@
-// put code here!
+var name = prompt ("What's your name"); alert ("Nice to meet you" + " " + name + "." + " " + "You just won a free vacation!");
+
+var destination = prompt ("Pick a destination: Fiji, Italy, South Africa (F,I,S) ");
+
+if (destination==="F") {
+  alert ("Great! Fiji has excellent beaches.");
+  var whoIsGoing = prompt ("How many people, including you, will be going?");
+
+        if (whoIsGoing < 3) {
```

Comment:
Watch your indentation! This says to me that `if (whoIsGoing < 3){` is inside `var whoIsGoing`... which is impossible!